### PR TITLE
Update go.mod and README with repo move to coreos org

### PR DESCRIPTION
**Pull request by sinnykumari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ Generates stream metadata for Fedora CoreOS using release metadata and overrides
...
+$ go get -u github.com/coreos/fedora-coreos-stream-generator
 ```
 Add the binary in your PATH variable. If you haven't done anything fancy, adding standard binary path where Go keeps binaries from package will be enough
 ```
 $ export PATH=$PATH:~/go/bin/
-$ fedora-coreos-stream-generator -release-index=<release index location for the stream>
+$ fedora-coreos-stream-generator -releases=<release index location for the stream> -output-file=stream.json
 ```
-Partial stream override can be given using option -override=<override json file location for the stream> . override.json need to be available locally to the system. https://github.com/coreos/fedora-coreos-streams repo will contains override for FCOS streams
+Partial stream override can be given using option -override=< /path/to/override.json> . override.json need to be available locally to the system. https://github.com/coreos/fedora-coreos-streams repo will contains override for FCOS streams
```
Comment (jlebon) [inline README.md]: Minor: looks like a stray space before `/path`? Also missing trailing periods a bit everywhere.

File: README.md [hunk 2]
```diff
@@ -4,22 +4,22 @@ Generates stream metadata for Fedora CoreOS using release metadata and overrides
 # Running
 Make sure you have latest go installed and then download the pacakge
```
Comment (jlebon) [inline README.md]: Minor: package

Comment (sinnykumari) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (sinnykumari) [reply README.md]: Fixed! Looks like I don't have habit of adding periods in the last sentence of a paragraph. <-- such as this one xD

**Review by jlebon — APPROVED** [review]
LGTM!

**Pull request merged**